### ISSUE TITLE
Unreal Engine 5.1 projects crash immediately after loading when plugin is enabled

### ISSUE THREAD

**Issue by Jawa1337** [microsoft/ProjectAcoustics#148]
Project builds ok but when i lauch it it immediately crashes with this error

LoginId:b3e5d49d40edd9fc958722bc9036ed83
EpicAccountId:3f8c74897f0e484a9cfaaa84300d93f1

Unhandled Exception: EXCEPTION_ACCESS_VIOLATION 0x00007ff94a9f2350

UnrealEditor_CoreUObject
UnrealEditor_CoreUObject
UnrealEditor_CoreUObject
UnrealEditor_Core
UnrealEditor_Core
UnrealEditor_Projects
UnrealEditor_Projects
UnrealEditor_Projects
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll

And in crash log it says it cannot load dll

Failed to load 'C:/Program Files/Epic Games/UE_5.1/Engine/Plugins/Marketplace/ProjectAcousticsNative/Binaries/Win64/UnrealEditor-ProjectAcoustics.dll' (GetLastError=1114)

[Mothership.log](https://github.com/microsoft/ProjectAcoustics/files/10420482/Mothership.log)

**Comment by NoelCross** [comment]
@Jawa1337 Unfortunately, there is not much information about the source of the failure in this report.  Perhaps try creating a Visual Studio solution file and execute the project in Visual Studio to see if we can get more callstack information.  

Does this same problem happen when using the UE5.1 sample scene for Project Acoustics (based on the Lyra scene)?

**Comment by Jawa1337** [comment]
@NoelCross Yes, if I enable plugin it happens with every project I try to open be it sample scene or completly new project
![error](https://user-images.githubusercontent.com/42612542/212747716-130e0977-55ce-47b1-881f-1c2b1e92e076.png)

**Comment by NoelCross** [comment]
@Jawa1337 I think that the best bet would be to recompile the plugin as a project plugin and see if you get better stack traces and logging.  I'm still not sure why you are seeing this, but recompiling might give us some better clues.
